docs/known_issues.rst

```diff
@@ -42,38 +40,38 @@ E.g.::
     >>> np.dot(q,q)
     285.0
     >>> np.hstack((q,q))
-    <Quantity [ 0., 1., 2., 3., 4., 5., 6., 7., 8., 9., 0., 1., 2., 3., 4.,
-                5., 6., 7., 8., 9.] (Unit not initialised)>
+    <Quantity [0., 1., 2., 3., 4., 5., 6., 7., 8., 9., 0., 1., 2., 3., 4., 5.,
+               6., 7., 8., 9.] (Unit not initialised)>
 
 ::
 
     >>> ratio = (3600 * u.s) / (1 * u.h)
     >>> ratio
-    <Quantity 3600.0 s / h>
+    <Quantity 3600. s / h>
     >>> np.array(ratio)
-    array(3600.0)
+    array(3600.)
     >>> np.array([ratio])
-    array([ 1.])
+    array([1.])
 
 Also in-place operations where the output is a normal `~numpy.ndarray`
```

Comment:
No need to skip. Rather, this whole part can be removed, since even in the minimum version of numpy we now support (>=1.13), this is no longer an issue!!